plugin/vim-todo-lists.vim

```diff
@@ -417,18 +431,18 @@ endfunction
 
 " Increases the indent level
 function! VimTodoListsIncreaseIndent()
-  normal! >>
+  normal! >>6l
 endfunction
 
 " Decreases the indent level
 function! VimTodoListsDecreaseIndent()
-  normal! <<
+  normal! <<6l
 endfunction
 
 " Migrates file to new format
 function! VimTodoListsMigrate()
   normal! mz
-  silent! execute ':%s/^\(\s*\)\(\[.\]\)/\1- \2/'
+  silent! execute ':%s/^\(\s*\)\(\[.\]\)/\1\* \2/'
```

Comment:
After switch to md-like syntax, syntax with `- [ ]` will be also deprecated, so:
```suggestion
  silent! execute ':%s/^\(\s*\)\(- \|\)\(\[.\]\)/\1\* \3/'
```